### DISCUSSION TITLE
Feature Request: Graphical Feedback for restoring backup

### DISCUSSION THREAD

**Discussion by cpyarger** [home-assistant/frontend#12339] [Supervisor]
When restoring from a backup, either on initial install, or from the backups menu, there is no feedback whatsoever showing that there is a restore going on, or that the system is rebooting.

In my opinion, It would be ideal to have something like the new updating window showing progress, with optional logs on what's going on, before letting the user know the system is about to reboot with a confirmation.

**Comment by bschatzow** [comment]
I opened a GitHub issue 

``` 
https://github.com/home-assistant/frontend/issues/11854#issuecomment-1050746877 
```
Stating that it is not just no feedback but the original screen reloads. Also on the restore my database doesn't start automatically and this causes recorded issues.
The issue on the screen has been confirmed but no one has been working on a fix.
Not sure how progress can be measured as communication between the HA device and the desktop is lost?
